brayns/common/loader/Loader.h

```diff
@@ -79,31 +80,50 @@ class Loader
 
     virtual ~Loader() = default;
 
+    /**
+     * @return The loaders supported file extensions
+     */
+    virtual std::vector<std::string> getSupportedExtensions() const = 0;
+
+    /**
+     * @return The loader name
+     */
+    virtual std::string getName() const = 0;
+
+    /**
+     * @return The loader properties
+     */
+    virtual PropertyMap getProperties() const = 0;
```

Comment:
pure virtual is not needed really, it's optional to have loader properties.

Reply:
I actually prefer having it pure virtual since that will force you to implement all expected methods in your loader. But if you wish I can remove it.

Reply:
Done.